libr/core/cmd_anal.c

```diff
@@ -4691,64 +4692,69 @@ static void cmd_anal_blocks(RCore *core, const char *input) {
 
 static void _anal_calls(RCore *core, ut64 addr, ut64 addr_end) {
 	RAnalOp op;
-	int bufi;
 	int depth = r_config_get_i (core->config, "anal.depth");
 	const int addrbytes = core->io->addrbytes;
 	const int bsz = 4096;
 	ut8 *buf;
-	ut8 *block;
-	bufi = 0;
+	ut8 *block0;
+	ut8 *block1;
+	int bufi = 0;
+	int bufi_max = bsz - 16;
 	if (addr_end - addr > UT32_MAX) {
 		return;
 	}
 	buf = malloc (bsz);
-	block = malloc (bsz);
-	if (!buf || !block) {
+	block0 = malloc (bsz);
```

Comment:
just `calloc` here? instead of doing `malloc` + `memset(.., 0, ..)`

Reply:
done